### PR TITLE
feat: add pre/postfix icons to autocomplete

### PR DESCRIPTION
**Pull request by NathanDeVuono**

### CODE REVIEW

File: src/components/press-autocomplete/press-autocomplete-ui.vue [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="ui input search">
+    <slot name="prefixIcon" />
```
Comment (ianwremmel) [inline src/components/press-autocomplete/press-autocomplete-ui.vue]: I think kebab case is more "Vuey"

Comment (NathanDeVuono) [reply src/components/press-autocomplete/press-autocomplete-ui.vue]: Its less Javascript-ey though 😞 I'll swap it out

Comment (bzang) [reply src/components/press-autocomplete/press-autocomplete-ui.vue]: you guys are gonna hate it when we make things "Rail-icious-y"

File: features/server/views/pages/autocomplete.ejs [hunk 2]
```diff
@@ -9,9 +9,15 @@
     <form class="ui form column" method="POST">
       <h3>Code Under Test</h3>
       <press-autocomplete name="autocomplete[input]" press-label-path="label" <%- value('autocomplete[input]') %>>
+        <template slot="prefixIcon">
+          <span class="prefix-icon">☃</span>
```
Comment (bzang) [inline features/server/views/pages/autocomplete.ejs]: image

File: features/server/views/partials/header.ejs [hunk 3]
```diff
@@ -8,4 +8,4 @@
       <div class="ui fluid card ud-page-section">
         <div class="content">
           <div class="ui grid">
-            <div class="ten wide computer ten wide large screen ten wide mobile ten wide tablet ten wide widescreen column">
+            <div>
```
Comment (NathanDeVuono) [inline features/server/views/partials/header.ejs]: These classes were causing CI to fail.  They caused the body to wrapped in a `width: 67.5% !important;` style and it was causing the submit button to wrap down onto another line and thus be hidden by the results pane which was failing the test

Comment (bzang) [reply features/server/views/partials/header.ejs]: image

File: features/server/views/pages/daterangepicker.ejs [hunk 4]
```diff
@@ -5,8 +5,8 @@
     <h2>Date Range Picker</h2>
   </header>
 
-  <main class="ui two column grid">
-    <form class="ui form column" method="POST">
+  <main class="">
+    <form class="" method="POST">
```
Comment (bzang) [inline features/server/views/pages/daterangepicker.ejs]: Should we get rid of these empty class attributes?

**Pull request merged**